controllers/provisioners/eks/helpers.go

```diff
@@ -350,34 +380,49 @@ func (ctx *EksInstanceGroupContext) GetLabelList() []string {
 		isOverride = true
 		overrideLabels := strings.Split(val, ",")
 		for _, label := range overrideLabels {
-			labelList = append(labelList, label)
+			keyVal := strings.Split(label, "=")
```

Comment:
what happens if label provided has no value? should have safety here around accessing `keyVal`?

Reply:
Added len check for this slice to ensure we set it to "" if there are not 2 values in the slice.

Reply:
mm, I don't see it - did you push it?

Reply:
Also, what if someone creates a malformed annotation (e.g. no equal sign)?

Reply:
Since this is user input we should be careful with out of bounds index

Reply:
strings.split will return a slice with the original string if the seperator does not exist.

Reply:
@backjo can you point me to the len check? 😄

Reply:
nvm, see it now 😓